app/components/CategoryOverviewView.js

```diff
@@ -3,10 +3,12 @@ import PropTypes from 'prop-types';
 import { View } from 'react-native';
 
 import CategoryHeaderAndroid from './CategoryHeader.android';
+import ProductList from '../containers/ProductList';
```

Comment:
@marchi-martius I see your point. But now I think that `CategoryOverviewView` doesn't give any UI. We just need it to hold next containers. So I would rather move mapping below `CategoryOverviewView`, not above.